crates/ty_python_semantic/resources/mdtest/import/star.md

```diff
@@ -1299,15 +1299,12 @@ reveal_type(Nope)  # revealed: Unknown
 ## `global` statements in non-global scopes
 
 Python allows `global` statements in function bodies to add new variables to the global scope, but
-we require a matching global binding or declaration. We lint on unresolved `global` statements, and
 we don't include the symbols they might define in `*` imports:
```

Comment:
We could change this (CPython does include these globals in `*` imports), but we'd need to make `ExportFinder` walk function and class bodies, which it currently doesn't. (And doesn't want to?)